project/RFDependenciesPlugin.scala

```diff
@@ -70,9 +71,9 @@ object RFDependenciesPlugin extends AutoPlugin {
       "jitpack" at "https://jitpack.io"
     ),
     // NB: Make sure to update the Spark version in pyrasterframes/python/setup.py
-    rfSparkVersion := "3.1.3",
-    rfGeoTrellisVersion := "3.6.1",
-    rfGeoMesaVersion := "3.2.0",
-    excludeDependencies += "log4j" % "log4j"
+    rfSparkVersion := "3.2.1",
+    rfGeoTrellisVersion := "3.6.3",
+    rfGeoMesaVersion := "3.4.1"
+    //excludeDependencies += "log4j" % "log4j"
```

Comment:
Wait, why was this excluded?

Reply:
I think it's another way to solve the CVE problem but idk!